OpenWeather/App/Search/SearchScreen.swift

```diff
@@ -17,71 +17,54 @@
 import OpenBytesNavigation
 import SwiftUI
 
-struct SearchScreen: View {
+struct LocationResultItem: View {
+    let name: String
+
     var body: some View {
-        List {
-            Button(
-                action: { Navigation.path.push(Date()) },
-                label: { Text("Push") }
-            )
+        VStack {
+            Text(name)
 
-            Button(
-                action: { Navigation.path.modal { Text("Modal") } },
-                label: { Text("Modal") }
-            )
+            // ...
+        }
+    }
+}
 
-            Button(
-                action: {
-                    Navigation.path.alert(
-                        title: Text("Alert Title"),
-                        primaryButton: .default(Text("Done")),
-                        secondaryButton: .cancel()
-                    )
-                },
-                label: { Text("Alert") }
-            )
+struct SearchScreen: View {
+    @State private var searchText: String = ""
+    @ObservedObject var viewModel: SearchViewModel
```

Comment:
Curious if we should adopt our new [ViewModel](https://github.com/0xOpenBytes/ViewModel) project.. 

Regardless, I think `searchText` should live inside the `ViewModel`.

Reply:
Some examples... https://github.com/0xOpenBytes/ViewModelExamples

Reply:
Yeah, I read about this `ViewModel` pattern, thinking to open this discussion with you, will study this option.